src/Console/Commands/BroadcastFaker/ChannelCheerFake.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Redbeed\OpenOverlay\Console\Commands\BroadcastFaker;
+
+class ChannelCheerFake extends Fake
```

Comment:
https://github.com/fzaninotto/Faker pls check this out, that would save you a lot of work

Reply:
https://github.com/redbeed/OpenOverlay/pull/1#discussion_r536857977